embassy-lora/src/lib.rs

```diff
@@ -1,12 +1,17 @@
 #![no_std]
 #![feature(async_fn_in_trait)]
+//#[cfg(feature = "stm32wl")]
+#![feature(try_blocks)]
 //! embassy-lora holds LoRa-specific functionality.
 
 pub(crate) mod fmt;
 
 /// interface variants required by the external lora physical layer crate (lora-phy)
 pub mod iv;
 
+//#[cfg(feature = "stm32wl")]
```

Comment:
this is probably not supposed to be commented

Reply:
Absolutely correct, fixed.